models/src/main/thrift/content/v1.thrift

```diff
@@ -544,9 +532,7 @@ struct InteractiveElementFields {
     9: optional string iframeUrl
     10: optional string role
     11: optional bool isMandatory
-    12: optional i32 height
-    13: optional i32 width
-    14: optional string sourceDomain
+    12: optional string sourceDomain
```

Comment:
In Thrift are you allowed to recycle a label like this?
Honestly don't know.

Reply:
Good spot tony: https://diwakergupta.github.io/thrift-missing-guide/

> Don’t change the numeric tags for any existing fields.